### PR TITLE
Improve defined properties documentation.

### PR DESCRIPTION
**Pull request by manthey**
This also removes some wrapping functions that are unnecessary.

### CODE REVIEW

File: src/imageTile.js [hunk 1]
```diff
@@ -1,117 +1,114 @@
...
 
-module.exports = (function () {
-  'use strict';
+/**
+ * This class defines a tile that is part of a standard "image pyramid", such
+ * as an open street map tile set.  Every tile is uniquely indexed by a row,
+ * column, and zoom level.  The number of rows/columns at zoom level z is
+ * `2^z`, the number of pixels per tile is configurable.
+ *
+ * By default, this class assumes that images are fetch from the url, but
```
Comment (johnkit) [inline src/imageTile.js]: FYI typo: images are fetch**ed** from the url.

File: src/osmLayer.js [hunk 2]
```diff
@@ -1,88 +1,85 @@
...
+var registry = require('./registry');
+var quadFeature = require('./quadFeature');
 
-  var $ = require('jquery');
-  var inherit = require('./inherit');
-  var tileLayer = require('./tileLayer');
-  var registry = require('./registry');
-  var quadFeature = require('./quadFeature');
+/**
+ * Create a new instance of osmLayer.  This is a [@link geo.tileLayer} with
```
Comment (johnkit) [inline src/osmLayer.js]: Is the @link a typo? I presume it should be _either_ regular brackets or curly brackets

**Comment by johnkit** [comment]
+1

* OK by me to merge without typos corrected
* There was some functional code in tileCache.js; although my qualifications as a reviewer are suspect, LGTM.

File: src/imageTile.js (hunk 1 above)
Comment (manthey) [reply src/imageTile.js]: Thanks.

File: src/osmLayer.js (hunk 2 above)
Comment (manthey) [reply src/osmLayer.js]: It should be braces.  Thanks.

File: src/fetchQueue.js [hunk 3]
```diff
@@ -1,235 +1,235 @@
...
-  'use strict';
-
-  var $ = require('jquery');
+var $ = require('jquery');
+
+/**
+ * This class implements a queue for Deferred objects.  Whenever one of the
+ * objects in the queue completes (resolved or rejected), another item in the
+ * queue is processed.  The number of concurrently processing items can be
+ * adjusted.  At this time (2015-12-29) most major browsers support 6
```
Comment (aashish24) [inline src/fetchQueue.js]: 2015-12-29? not sure which date it needs to refer to. Just making a check here

File: src/fetchQueue.js [hunk 4]
```diff
@@ -1,235 +1,235 @@
...
+ * @class
+ * @alias geo.fetchQueue
+ *
+ * @param {object} [options] A configuration object for the queue.
+ * @param {number} [options.size=6] The maximum number of concurrent deferred
+ *    objects.
+ * @param {number} [options.track=600] The number of objects that are tracked
+ *    that trigger checking if any of them have been abandoned.  The fetch
+ *    queue can grow to the greater of this size and the number of items that
+ *    are still needed.  Setting this to a low number will increase
```
Comment (aashish24) [inline src/fetchQueue.js]: I think this sentence could be improved further .

Comment (manthey) [reply src/fetchQueue.js]: Suggestions welcome.  Though this wasn't altered by this PR, so such changes could be a different PR.

**Pull request merged**